haas/api.py

```diff
@@ -796,20 +796,22 @@ def port_detach_nic(switch, port):
     port.nic = None
     db.session.commit()
 
-
-@rest_call('GET', '/free_nodes', Schema({}))
-def list_free_nodes():
-    """List all nodes not in any project.
+@rest_call('GET', '/node/<is_free>', Schema({'is_free' : basestring}))
```

Comment:
I checked online and people use both plural and singular forms equally. I prefer idea of a resource name without getting into "English" of its plurality. As it is, many plural words do not have well defined patterns. child - children, foot -feet, sheep -sheep. So to be consistent and avoid any such problems in the future, lets stick with the singular form of resources. Whatever api is plural should be changed as we go.